i2c-ch341-usb.c

```diff
@@ -1047,8 +1048,8 @@ static int ch341_gpio_probe (struct ch341_device* ch341_dev)
     for (i = 0; i < CH341_GPIO_NUM_PINS; i++)
     {
         // in case the pin as CS signal, it is an GPIO pin
-        if ((result = gpio_request(gpio->base + j, ch341_board_config[i].name)) ||
-            (result = gpio_export (gpio->base + j, ch341_board_config[i].pin != 21 ? true : false)))
+        if ((result = gpio_request(gpio->base + j, ch341_board_config[i].name))/* ||
+            (result = gpio_export (gpio->base + j, ch341_board_config[i].pin != 21 ? true : false))*/)
```

Comment:
`gpio_export()` seems to be removed in linux, I don't know how to fix it so I just commented it out for now

My CH341 only has I2C exposed so I can't really test it.